app/src/main/java/com/getsensibill/spendmanager/demo/fragment/FragmentJavaActivity.java

```diff
@@ -0,0 +1,96 @@
+package com.getsensibill.spendmanager.demo.fragment;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.FragmentTransaction;
+
+import android.os.Bundle;
+
+import com.getsensibill.spendmanager.demo.R;
+import com.getsensibill.spendmanager.demo.databinding.ActivityFragmentJavaBinding;
+import com.getsensibill.web.data.UiFinishReason;
+import com.getsensibill.web.data.configuration.NavigationIntent;
+import com.getsensibill.web.data.configuration.ProgrammaticTheme;
+import com.getsensibill.web.data.models.Brand;
+import com.getsensibill.web.ui.WebUiFragment;
+import com.getsensibill.web.ui.WebUiNetworkErrorFragment;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Directly using the [WebUiFragment]
+ * <p>
+ * When using [WebUiFragment] directly, please ensure to properly implement the [Listener] methods.
+ * Also, please use, or see [WebUiNetworkErrorFragment] as reference for what to display when
+ * [Listener.onDisplayNetworkError] is called.
+ */
+public class FragmentJavaActivity extends AppCompatActivity implements WebUiFragment.Listener {
+    private ActivityFragmentJavaBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        binding = ActivityFragmentJavaBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+
+        if(savedInstanceState == null) {
+            WebUiFragment fragment = new WebUiFragment();
+            Bundle bundle = fragment.getArguments();
+            if (bundle == null) {
+                bundle = new Bundle();
+            }
+
+            // Pass in a navigation override. Defaults as .DASHBOARD
+            bundle.putParcelable(
+                    WebUiFragment.ARG_NAVIGATION_OVERRIDE,
+                    NavigationIntent.DASHBOARD.INSTANCE
+            );
+
+            // Pass in a custom Theme override. Defaults to null
+            ProgrammaticTheme theme = new ProgrammaticTheme(new Brand());
+            bundle.putParcelable(WebUiFragment.ARG_PROGRAMMATIC_THEME_OVERRIDE, theme);
+
+            fragment.setArguments(bundle);
+
+            FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+            transaction.replace(R.id.fragment_container, fragment, "FRAGMENT_TAG");
+            transaction.commit();
+        }
+    }
+
+    /**
+     * Required Override when using [WebUiFragment.Listener]
+     *
+     * Handle when the network error fragment needs to be navigated to. We have provided a
+     * [WebUiNetworkErrorFragment] that you can display to the user. Or you can use your
+     * own fragment or even an alternative logic flow if desired.
+     */
+    @Override
+    public void onDisplayNetworkError(boolean b) {
```

Comment:
Did we lose the parameter name here?

Reply:
ya weird. when I use the intellescence to make it, it comes out like that. Updated.